notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1881 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cop_sampled["cop_class"] = cop_sampled["discrete_classification"].apply(lambda x: map_values(x, 40))
</pre><p> </p><p>nit: a one liner instead of map_values here is probably easier to understand</p><p><code>lambda x: 1 if x == 40 else 0</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>